마크다운 리스트 깊이 조절 기능 추가

RBJH opened the pull request:
state의 options에 depth 추가
tab 및 shift+tab 시 리스트 깊이 조절 가능

RBJH committed:
Merge: branch 'develop' of https://github.com/connect-foundation/2019-14 into feature/list

RBJH committed:
Fix: 마크다운 리스트 셀 깊이 단계 조절 가능

기존 마크다운 리스트의 깊이 조절이 안되는 부분을
options에 depth 변수를 통해 깊이 조절이 가능하게 수정

ImuruKevol commented on client/src/components/editor/cells/List/index.jsx:
```diff
@@ -87,22 +91,48 @@ const ListCell = ({ cellUuid }) => {
+    console.log("tab event:", options[cellIndex]);
+  };
+
+  const shiftTabEvent = (ev) => {
```
깊이가 있을 때 커서가 맨 앞에서 백스페이스를 눌렀을 때 깊이가 1씩 감소하도록 처리해주세요!
그리고 깊이가 없을 때 처리도요!
커서 위치가 start: 0, end: 0일 때 shiftTabEvent를 실행시키도록 하면 될 것 같습니다.

dimsssss reviewed (commented):
고생하셨습니다

RBJH committed:
Merge: branch 'develop' into feature/list

RBJH committed:
Fix: 마크다운 깊이 조정 시 백스페이스 이벤트 수정

기존 마크다운 깊이가 있을 때 맨 앞의 커서에서 백스페이스를 누르면
리스트가 기본 input 창으로 바뀌었는데, 코드 리뷰 간 리스트의 깊이를
줄여야 하는 쪽으로 합의가 되어 backspace 이벤트 수정
